spaces/domains.json

```
@@ -169,5 +169,6 @@
   "vote.akropolis.io": "akropolis.eth",
   "vote.blockzerolabs.io": "blockzerolabs.eth",
   "mscore.bprotocol.org": "m.bsnapshot.eth",
-  "cscore.bprotocol.org": "bsnapshot.eth"
+  "cscore.bprotocol.org": "bsnapshot.eth",
+  "vote.supra.finance": "suprafinance"
```

Your domain field value should be similar to your space directory `"my.custom.domain": "my-space.eth"`.
```suggestion
  "vote.supra.finance": "suprafinance.xyz"
```